experimental/adaptive-dialog/docs/memory-model-overview.md

```diff
@@ -9,7 +9,15 @@ As with web apps, a bot is inherently stateless; a different instance of your bo
 
 All memory properties, in all memory scopes, are property bags, meaning you can store arbitrary information on them.
 
-See [here][3] for guidance on when to use each scope
+See [here][3] for guidance on when to use each scope. 
+
+Here are the different memory scopes available - 
+- [User](#User-scope)
+- [Conversation](#Conversation-scope)
+- [Dialog](#Dialog-scope)
+- [Turn](#Turn-scope)
+- [Settings](#Settings-scope)
+- [Short-hands](#Memory-short-hand-notations)
 
```

Comment:
@vishwacsena , what about custom memory scope defined by users? Or to be specific in relation to Composer, ability to bring your own memory scope?

Reply:
I do not have any samples showcasing that. Recommend we add that in composer docs.